Sources/TabPageOption.swift

```diff
@@ -8,6 +8,13 @@
 
 import UIKit
 
+public enum ScrollViewContent {
+    case None
```

Comment:
Please write in lowercase letters
https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Enumerations.html